src/coreclr/tools/aot/crossgen2/Crossgen2RootCommand.cs

```diff
@@ -39,6 +39,8 @@ internal class Crossgen2RootCommand : CliRootCommand
             new("--optimize-space", "--Os") { Description = SR.OptimizeSpaceOption };
         public CliOption<bool> OptimizeTime { get; } =
             new("--optimize-time", "--Ot") { Description = SR.OptimizeSpeedOption };
+        public CliOption<bool> EnableCachedInterfaceDispatchSupport { get; } =
```

Comment:
Based on the other options here this should be `--cid` not `--CID`, though I'm not sure whether the parsing is case sensitive anyway